docs/services/pppoe-server.rst

```diff
@@ -209,6 +209,36 @@ is working.
 Other attributes can be used, but they have to be in one of the dictionaries
 in /usr/share/accel-ppp/radius.
 
+
+Load Balancing
+--------------
+
+For balancing sessions on PPPoE servers exist special mechanisms based on delay PADO packets.
+
+.. code-block:: none
+
+  set service pppoe-server pado-delay 50 sessions '500'
+  set service pppoe-server pado-delay 100 sessions '1000'
+  set service pppoe-server pado-delay 300 sessions '3000'
+
+First 499 session connects without delay. PADO packets will be delayed 50 ms for connection from 500 to 999, this trick allows other PPPoE servers send PADO faster and clients will connect for other servers. Last command says that this PPPoE server can serve only 3000 clients.
+
+
+Maintenance mode
+----------------
+
+For maintenance mode implemented commands which allow keep already established connections and don't allow new clients connections. After the last client is disconnected from the server, then PPPoE daemon stopped and maintenance can start without problem for clients connections.
+
+.. code-block:: none
+
+  vyos@vyos:~$ set pppoe-server maintenance-mode enable
+
+Maintenance mode also possible to cancel with following command
```

Comment:
Maintenance mode _is_ also possible to cancel...
Better yet: You can also cancel maintenance mode with the following command: